src/errors.rs

```diff
@@ -1,31 +1,82 @@
 use crate::network::{ActorId, Attribute, ObjectId, StreamId, Trajectory};
 use std::str;
-
-#[derive(PartialEq, Debug, Clone, Fail)]
+use std::error::Error;
+use std::fmt::{Display, Formatter};
+use std::fmt;
+
+//#[derive(PartialEq, Debug, Clone)]
```

Comment:
Are you keeping this commented out error around for something?

Reply:
Nope! Good bye `BoxcarError`. :wave: